app/src/ui/branches/pull-request-list.tsx

```diff
@@ -27,6 +27,9 @@ interface IPullRequestListProps {
   /** The pull requests to display. */
   readonly pullRequests: ReadonlyArray<PullRequest>
 
+  /** The pull request associated with the current branch. */
```

Comment:
Could we please add something like 'this is used to pre-select the currently checked out PR in the list of pull requests' to make it a little more descriptive?